cloudvolume/lib.py

```diff
@@ -154,7 +154,7 @@ def extract_path(cloudpath):
 def toabs(path):
   home = os.path.join(os.environ['HOME'], '')
   # remove file protocol
-  path = re.sub('^file://?', '', path)
+  # path = re.sub('^file://?', '', path)
```

Comment:
Is this intended to be commented out?

Reply:
yes, originally there is no such removal. I should delete this rather than commented out.